scripts/pr_review_test.py

```
@@ -1265,6 +1267,569 @@ def test_every_secret_the_guide_requires_is_one_the_workflow_accepts(self) -> No
+        binding = pr_review.PullBinding("a" * 40, "b" * 40, "c" * 40, pr_review.RUBRIC_VERSION)
+        seen = pr_review.Finding("high", "f.go", 1, "Old problem", "w", "f")
+        marker = {
+            "state": "findings",
+            "identity": "old:old:old:x",
+            "mode": "deep",
+            "findings": pr_review.finding_fingerprint(seen),
+            "html_url": "u",
+        }
+        published: dict[str, str] = {}
+
+        def capture(_repo, comment_id, _token, body, _corr):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `capture`

(ANN202)